web-local/src/common/errors/ErrorMessages.ts

```diff
@@ -12,6 +12,8 @@ export const ExplorerSourceModelIsInvalid = "Model query has errors.";
 export const ExplorerTimeDimensionDoesntExist =
   "Previously selected timestamp column does not exist.";
 export const ExplorerSourceColumnDoesntExist = "not found in FROM clause!"; // the full DuckDB error message is `Binder Error: Referenced column "COLUMN_NAME" not found in FROM clause!`
+export const ExplorerDashboardDoesntExist =
+  "Cannot read properties of undefined (reading 'id')";
```

Comment:
Please update the backend code to handle this instead of checking for reference error.